### PR TITLE
Issue #616 - Fix Pantheon deploy error

### PR DESCRIPTION
**Pull request by mrdavidburns**
fixes: #616 

This patch created by @alexanderpatriciop has been tested on multiple projects and does complete the deployment without failure

### CODE REVIEW

File: tasks/drupal.yml [hunk 1]
```diff
@@ -68,7 +68,8 @@ tasks:
       - ./vendor/bin/drush {{.site}} --yes cache:rebuild
       - ./vendor/bin/drush {{.site}} --yes deploy:hook
       - |
-        if [[ $(./vendor/bin/drush {{.site}} config:status --format=json --state=Different) != '[]' ]]; then
+        config_status_output=$(./vendor/bin/drush {{.site}} config:status --format=json --state=Different | tail -n1)
```
Comment (deviantintegral) [inline tasks/drupal.yml:71]: So the only real difference here is `tail -n1`? Is drush outputting some text beyond the json output? I would have expected any non-json text to be in stderr, not stdout.

Comment (mrdavidburns) [reply tasks/drupal.yml:71]: From what I've seen by running the commands myself the last characters are `[]`. However, we have merged this patch on a project and it completes without error.

See my comment here about the output: https://github.com/Lullabot/drainpipe/issues/616#issuecomment-2243987592

Comment (penyaskito) [reply tasks/drupal.yml:71]:
```suggestion
       # drush config:status --format=json is outputting notices in Pantheon even with the json format, 
       # so we need to tail the last line.
        config_status_output=$(./vendor/bin/drush {{.site}} config:status --format=json --state=Different | tail -n1)
```
Let's at least add a brief comment for this.
This has worked for our project dev deployments, which were failing before.

Comment (penyaskito) [reply tasks/drupal.yml:73]: @deviantintegral This is executed with command substitution as ' $(./vendor/bin/drush {{.site}} config:status --format=json --state=Different)`
As far as I know that doesn't include stderr, but I'm not 100% sure.

Comment (deviantintegral) [reply tasks/drupal.yml:73]: It doesn't!

```
$ ls $(echo composer.lock 1>&2)
composer.lock
README.md     composer.json  config             node_modules  phpcs.xml.dist  postcss.config.js  scripts             sites.json  vendor  yarn.lock
Taskfile.yml  composer.lock  eslint.config.mjs  package.json  phpstan.neon    renovate.json      sites-domains.json  test        web
```

If it did, the above would have only shown composer.lock. Makes sense, thanks for the TIL'ed!

**Review by deviantintegral — APPROVED** [review]

**Pull request merged**